gh-badges/package.json

```diff
@@ -35,10 +35,8 @@
   },
```

Comment:
Couldn't get this up above on the `name` field, but where did we land on potentially renaming the package (or is that out of scope for this PR)?

Reply:
Yep - changing the name is one of the things to do for version 3. Its one of the points in #3587
The scope of this PR is just to remove doT, as opposed to completing _all_ the remaining breaking changes for V3.